readme.md

```diff
@@ -156,3 +159,32 @@ Exposed for instance checking and sub-classing.
 - [p-min-delay](https://github.com/sindresorhus/p-min-delay) - Delay a promise a minimum amount of time
 - [p-retry](https://github.com/sindresorhus/p-retry) - Retry a promise-returning function
 - [More…](https://github.com/sindresorhus/promise-fun)
+
+## AbortSignal
+
+> Modern alternative to `p-timeout`
+
+Asynchronous functions like `fetch` can accept an [AbortSignal](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal), which can be conveniently created with [AbortSignal.timeout()](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal/timeout_static).
+
+The advantage over `p-timeout` is that the promise-generating function (like `fetch`) is actually notified that the user is no longer expecting an answer, so it can interrupt its work and free resources.
+
+```js
+// Call API, timeout after 5 seconds
+const response = await fetch('./my-api', {signal: AbortSignal.timeout(5000)});
+```
+
+```js
+async function buildWall(signal) {
+	for (const brick of bricks) {
+		signal.throwIfAborted();
+		// Or: if (signal.aborted) return;
+
+		await layBrick();
+	}
+}
+
+// Stop long work after 60 seconds
+await buildWall(AbortSignal.timeout(60_000))
+```
+
+You can also [combine multiple signals](https://github.com/fregante/abort-utils/blob/main/source/merge-signals.md), like when you have a timeout _and_ an `AbortController` triggered with a Cancel button click.
```

Comment:
I'd link to `AbortSignal.any()` directly but it's Chrome-only for now (and Node 20+)

Reply:
Can you link to both? I'd prefer to have the built-in one as the main recommendation and you can mention that for backward compatibility, use this package.